frontend/javascripts/oxalis/model/sagas/automatic_brush_saga.js

```diff
@@ -0,0 +1,255 @@
+// @flow
+import * as tf from "@tensorflow/tfjs";
+import memoizeOne from "memoize-one";
+
+import { type Saga, call, fork, select, take, _cancel } from "oxalis/model/sagas/effect-generators";
+import floodfill from "libs/floodfill";
+import { FlycamActions } from "oxalis/model/actions/flycam_actions";
+import { type InferSegmentationInViewportAction } from "oxalis/model/actions/volumetracing_actions";
+import { sleep } from "libs/utils";
+import { type Vector2, type Vector3, type OrthoView, OrthoViews } from "oxalis/constants";
+import { getMeanAndStdDevFromDataset } from "admin/admin_rest_api";
+import type { APIDataset } from "admin/api_flow_types";
+import { createWorker } from "oxalis/workers/comlink_wrapper";
+import TensorFlowWorker from "oxalis/workers/tensorflow.worker";
+import mainThreadPredict from "oxalis/workers/tensorflow.impl";
+import ThreeDMap from "libs/ThreeDMap";
+import Model from "oxalis/model";
+import Toast from "libs/toast";
+import { enforceVolumeTracing } from "oxalis/model/accessors/volumetracing_accessor";
+import Dimensions from "oxalis/model/dimensions";
+import api from "oxalis/api/internal_api";
+import {
+  getPosition,
+  getPlaneExtentInVoxelFromStore,
+} from "oxalis/model/accessors/flycam_accessor";
+import { V2 } from "libs/mjs";
+import { getBaseVoxelFactors } from "oxalis/model/scaleinfo";
+
+const outputExtent = 100;
+const inputContextExtent = 20;
+const inputExtent = outputExtent + 2 * inputContextExtent;
+const NUM_PREDICT_SLICES = 3;
+const FLOODFILL_THRESHOLD = 0.7;
```

Comment:
Could you change this to `const FLOODFILL_THRESHOLD = window.floodfillThreshold || 0.7` please? Then, we could fine tune this ad hoc for demo purposes :)